### PR TITLE
Fixes body IDs selection when computing `feet_slide` reward for locomotion-velocity task

### PR DESCRIPTION
**Pull request by dtc103**
# Description

Since I want to use Isaac Lab for training a Unitree Go2, I was playing around with the examples to get used to the framwork. While playing around, I got the following error message:

```
reward = torch.sum(body_vel.norm(dim=-1) * contacts, dim=1)
RuntimeError: The size of tensor a (19) must match the size of tensor b (4) at non-singleton dimension 1
```
The reward term was added as follows:
``` 
sliding_feet = RewTerm(
        func=mdp.feet_slide,
        params={"sensor_cfg": SceneEntityCfg("contact_forces", body_names=".*foot")},
        weight=0.1
    )
``` 

After some code investigation, I found out, that inside the feet_slide function the body velocities are queried as "asset.data.body_lin_vel_w[:, asset_cfg.body_ids, :2]". This would return the velocity of all body parts, since asset_cfg.body_ids contains the ids of all body parts.  
Therefore we need to change the line to "body_vel = asset.data.body_lin_vel_w[:, sensor_cfg.body_ids, :2]" since we only want the velocity of the body parts that contain the force sensors.  
This means we have to change `asset_cfg.body_ids` to `sensor_cfg.body_ids` inside the tensor call

Doing this lead to successfully running the simulation without failure.   

No additional dependencies are necessary for this fix.  

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshots

Please attach before and after screenshots of the change if applicable.

| Before | After |
| ------ | ----- |
| fix_before | fix_after

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x]  I have added my name to the CONTRIBUTORS.md or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab_tasks/config/extension.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 
 # Note: Semantic Versioning is used: https://semver.org/
-version = "0.10.0"
+version = "0.10.1"
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab_tasks/config/extension.toml]: Could you also update the `CHANGELOG.rst` file?

**Review by Mayankm96 — APPROVED** [review]
Thanks a lot for the fix! Could you also adapt the CHANGELOG file?

File: source/extensions/omni.isaac.lab_tasks/config/extension.toml (hunk 1 above)
Comment (dtc103) [reply source/extensions/omni.isaac.lab_tasks/config/extension.toml]: Changed it :D

File: source/extensions/omni.isaac.lab_tasks/docs/CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 
+0.10.1 (2024-10-20)
+~~~~~~~~~~~~~~~~~~~
+
+Fixed
+^^^^^
+
+* Fixed the wrong selection of body id's in the ``feet_slide`` reward function in omni.isaac.lab_tasks.manager_based.locomotion.velocity.mdp.rewards.
+  Changed parameter from ``asset_cfg.body_ids`` to ``sensor_cfg.body_ids``.
+
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab_tasks/docs/CHANGELOG.rst]:
```suggestion
* Fixed the wrong selection of body id's in the :meth:`omni.isaac.lab_tasks.manager_based.locomotion.velocity.mdp.rewards.feet_slide`
  reward function. This makes sure the right IDs are selected for the bodies.


```

**Pull request merged**